ci-operator/step-registry/optional-operators/configure-proxy-registry/optional-operators-configure-proxy-registry-commands.sh

```diff
@@ -40,6 +40,17 @@ echo "Configuring proxy registry : \"$OO_CONFIGURE_PROXY_REGISTRY\""
 # step-3: Disable the default OperatorSources/Sources (for redhat-operators, certified-operators, and community-operators) on your 4.5 cluster (or default CatalogSources in 4.6+) with the following command:
 oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]'
 
+# If ENABLE_REDHAT_OPERATOR_CATALOG is true, enable redhat-operators catalog
+if [[ "${ENABLE_REDHAT_OPERATOR_CATALOG}" == "true" ]]; then
+    oc patch OperatorHub cluster --type merge --patch '{"spec":{"sources":[{"disabled": false,"name": "redhat-operators"}]}}'
+    if [[ "$?" == 0 ]] && oc get catsrc -n openshift-marketplace | grep -q redhat-operators; then
```

Comment:
we will probably not get to this line in case of failure from  previous command (oc patch)
The strict options are set at the beginning of this file: `set -o errexit`

Reply:
@lslebodn thanks for the suggestion! I removed checking the status of the previous command: `$?==0`. 
I kept the check if the redhat-catalog was actually enabled, I think it is good to have this extra check, please correct me if I'm wrong.
Also, CVP-3069 is on-hold currently, so I'm keeping this PR as a WIP in case we need it in the future.